src/js/media-time-range.js

```diff
@@ -34,13 +34,20 @@ template.innerHTML = `
       color: #fff;
     }
 
-    [part~="box"] {
-      display: flex;
-      flex-direction: column;
-      align-items: center;
+    #preview-rail,
+    #current-rail {
+      width: 100%;
```

Comment:
this is a key part the "rails" are expanded to 100% width which equals the width of the parent and the input range.
then we set CSS transform translateX in percentages which are percentages based on the element itself not the parent. transforms are weird that way